crates/edr_eth/src/transaction.rs

```diff
@@ -234,6 +234,11 @@ pub trait IsEip4844 {
     fn is_eip4844(&self) -> bool;
 }
 
+pub trait IsExecutable {
```

Comment:
Maybe we could add more clarification here for the trait doc comment itself

Reply:
Addressed in https://github.com/NomicFoundation/edr/pull/671/commits/9d3adad2a37095385ca9a6f7ff5c9c719e842f2c